examples/plotting/hmi_LOSmag.py

```diff
@@ -0,0 +1,53 @@
+"""
+====================
+Creating a HMI image
+====================
+
+In this example, we will demonstrate how to download and plot
+line of sight magnetic field data from HMI.
+"""
+# Start by importing the necessary modules.
+import sunpy.map
+from sunpy.net import Fido, attrs as a
+import astropy.units as u
```

Comment:
these imports are in the wrong order, astropy should be above sunpy with a blank line in between.